taskt/Core/Automation/Commands/Word Commands/WordReplaceTextCommand.cs

```diff
@@ -61,7 +61,7 @@ public override void RunCommand(object sender)
             var wordObject = engine.GetAppInstance(vInstance);
```

Comment:
Please update a comment on line-60 to "//get word app object"

Reply:
done